books/free-programming-books-pt_BR.md

```diff
@@ -45,11 +46,25 @@
 * [RegEx](#regex)
 * [Ruby](#ruby)
 * [Shell Script](#shell)
-* [Vim](#vim)
 
 
 ### Agnósticos
 
+#### IDE / Editores
+
+* [Visual Studio Code: Produtividade infinita](https://github.com/bylearn/VS-Code-Produtividade-Infinita) - Felipe Cabrera Ribeiro dos Santos
+
+
+#### Vim
+
+* [Editor VI - Guia de Referência](https://aurelio.net/curso/material/vim-ref.html) - Aurelio Marinho Jargas
+* [O Editor de Texto Vim](https://code.google.com/p/vimbook) - Sérgio Luiz Araújo Silva et al.
+* [Vim Avançado](https://aurelio.net/vim/vim-avancado.txt) - Aurelio Marinho Jargas
```

Comment:
this may be a cheatsheet?

Reply:
also a cheatsheet

Reply:
Ok, I'll move it 😉

It was an improving changing http to https

Reply:
Hmmm. No problem putting them on the cheatsheet list; IIRC there are already some non-english entries there.

Reply:
Done. Check it now again